pkg/imgpkg/bundle/bundle.go

```diff
@@ -19,48 +21,116 @@ const (
 )
 
 type Bundle struct {
-	plainImg     *plainimg.PlainImage
-	imgRetriever ctlimg.ImagesMetadata
+	plainImg      *plainimg.PlainImage
+	imgRetriever  ctlimg.ImagesMetadata
+	numSubBundles int
```

Comment:
numSubBundles variable seems to be only used in pull. let's localize it. if we need to use in recursion, it could be passed around.

Reply:
Ok. done